pymongo/pool.py

```diff
@@ -590,7 +590,7 @@ def unpin(self):
 
     def hello_cmd(self):
         # Handshake spec requires us to use OP_MSG+hello command for the
-        # initial handshake in load balanced or versioned api mode.
```

Comment:
@behackett should I be changing comments? They are not user facing.

Reply:
Might as well to avoid confusion from future maintainers.

Reply:
Sounds good.